src/bitcoin-chainstate.cpp

```diff
@@ -19,7 +19,7 @@ using node::fReindex;
 using node::LoadChainstate;
 using node::VerifyLoadedChainstate;
 
-const extern std::function<std::string(const char*)> G_TRANSLATION_FUN = nullptr;
+__attribute__ ((visibility("default"))) const extern std::function<std::string(const char*)> G_TRANSLATION_FUN = nullptr;
```

Comment:
In commit "f" (e5579f11b702ad978cd4f672b362e6346c699234)

I don't think `extern` belongs here. Maybe if you drop the `extern` you could drop the visibility attribute as well?

Reply:
You're right, I've removed it.

However, we still need the visibility attribute for the reasons detailed in the commit message for 7f7f92f81d06b0a3888fad102d3bc09a64483995